### DISCUSSION TITLE
Enable use GuiLab Xaml Parser

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#13433] [Ideas]
In PR https://github.com/kekekeks/XamlX/pull/91 allows you to use  [GuiLab Xml Parser](https://github.com/KirillOsenkov/XmlParser) in `XamlC` , which has a fault tolerance.
I would like to propose that you also enable support for Avalonia.

**Comment by kekekeks** [comment]
Note that said parser is supposed to be used for IDE plugins, it doesn't pass all the checks required for proper whitespace handling

**Comment by workgroupengineering** [comment]
> it doesn't pass all the checks required for proper whitespace handling

I am aware of it. I partially solved it, as soon as it's ready I'll make a PR on the main GuiLabs Parser repository
